imap_processing/codice/cdf_attrs.py

```diff
@@ -44,14 +44,14 @@
 
 codice_l1a_global_attrs = GlobalDataLevelAttrs(
     data_type="L1A->Level-1A",
-    logical_source="imap_codice_l1a",
+    logical_source="imap_codice_l1a_sci",
     logical_source_desc="IMAP Mission CoDICE Instrument Level-1A Data",
     instrument_base=codice_base,
 )
 
 codice_l1b_global_attrs = GlobalDataLevelAttrs(
     data_type="L1B->Level-1B",
-    logical_source="imap_cpdice_l1b",
+    logical_source="imap_cpdice_l1b_sci",
```

Comment:
is that a typo?
```suggestion
    logical_source="imap_codice_l1b_sci",
```

Reply:
good catch, I was only updating the latter thing and not looking at the text before the "sci" portion! Fixed now.